Kitodo/src/main/java/org/kitodo/services/data/TemplateService.java

```diff
@@ -118,7 +130,8 @@ private BoolQueryBuilder readFilters(Map<String, String> filterMap) throws DataE
         BoolQueryBuilder query = null;
 
         for (Map.Entry<String, String> entry : filterMap.entrySet()) {
-            query = serviceManager.getFilterService().queryBuilder(entry.getValue(), ObjectType.PROCESS, true, false, false);
+            //query = serviceManager.getFilterService().queryBuilder(entry.getValue(), ObjectType.TEMPLATE, false,
```

Comment:
Why are you commenting this out?

Reply:
This is possible, but you open a null pointer exception a few lines later (line 123/125 ... `query.must(...);`) as query object is not initialised.